src/OpenTelemetry.Instrumentation.Process/README.md

```diff
@@ -108,11 +124,11 @@ The number of processors (CPU cores) available to the current process.
 The API used to retrieve the value is [System.Environment.ProcessorCount](https://learn.microsoft.com/dotnet/api/system.environment.processorcount).
 
 > **Note**
-> This metric is under [discussion][2] and not part of the
-[Process Metrics Spec][3] at this time.
+> This metric is under [discussion][6] and not part of the
+[Process Metrics Spec][7] at this time.
 
-[2]: https://github.com/open-telemetry/opentelemetry-specification/issues/3200
-[3]: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/process-metrics.md
+[6]: https://github.com/open-telemetry/opentelemetry-specification/issues/3200
```

Comment:
I vote for avoiding this pattern since it requires updating the numbers each time a new link is inserted.

Reply:
Yeah, I kind of see the annoying part of it while adding more references to the file.
Originally, I think it is a good way to get away with the markdown lint check of 80 chars per line.

Reply:
_get away with the markdown lint check of 80 chars per line._

We can fix it with other ways. Check how we handle it in otel .net repo.

Reply:
moved away from the reference link syntax